src/components/Markdown/Markdown.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box } from "@chakra-ui/react";
+import ChakraUIRenderer from "chakra-ui-markdown-renderer";
+import { FunctionComponent } from "react";
+import ReactMarkdown from "react-markdown";
+import rehypeRaw from "rehype-raw";
+import { Code } from "./Code";
+import { Headings } from "./Headings";
+import { Hr } from "./Hr";
+import { Ul, Ol, Li } from "./List";
+import { A, Blockquote, Em, P, Pre, Sup } from "./Text";
+
+const components = ChakraUIRenderer(
```

Comment:
Are we fully overriding everything? If so I think we can remove this renderer all together.

Reply:
I'll check, I've left it in to be safe, but there's always an edge case 🙃

Reply:
Ok it's good 2 remove